.pre-commit-config.yaml

```diff
@@ -7,7 +7,6 @@ repos:
     -   id: check-yaml
     -   id: check-json
     -   id: check-toml
-    -   id: check-added-large-files
```

Comment:
Didn't find a way to change the threshold from 500KB, which is smaller than the svgs used in testing.